admin/services/check-window.service.js

```diff
@@ -114,6 +93,88 @@ const checkWindowService = {
     const activeCheckWindow = await checkWindowDataService.sqlFindOneActiveCheckWindow(currentCheck.checkWindow_id)
     if (!activeCheckWindow) throw new Error('There is no open check window')
     return activeCheckWindow
+  },
+
+  /**
+   * Get all check windows
+   * @param {String} sortField
+   * @param {String} sortDirection
+   * @returns {Object}
+   */
+  getAllCheckWindows: async(sortField, sortDirection) => {
+    // fetch past current and future check windows
+    const checkWindows = await checkWindowDataService.sqlFindAllCheckWindows(sortField, sortDirection)
+    const pastCheckWindows = []
+    const newCheckWindows = []
+    checkWindows.map(cw => {
+      const adminStartDate = moment(cw.adminStartDate)
+      const checkStartDate = moment(cw.checkStartDate)
+      const checkEndDate = moment(cw.checkEndDate)
+
+      if (moment(checkStartDate).isAfter(moment(checkEndDate))) {
```

Comment:
Aren't `checkStartDate` and `checkEndDate` already moment objects?  Why clone them?

Reply:
@jon-shipley updated